Makefile

```diff
@@ -16,21 +16,33 @@ IMAGE_REGISTRY?=registry.svc.ci.openshift.org
 
 # Include the library makefile
 include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
-        golang.mk \
-        targets/openshift/deps-gomod.mk \
-        targets/openshift/images.mk \
-        targets/openshift/bindata.mk \
-        targets/openshift/codegen.mk \
+	golang.mk \
+	targets/openshift/deps-gomod.mk \
+	targets/openshift/images.mk \
+	targets/openshift/bindata.mk \
+	targets/openshift/codegen.mk \
 )
 
+# Codegen module needs setting these required variables
+PACKAGE_BASE := github.com/ovirt/csi-driver-operator
+CODEGEN_OUTPUT_PACKAGE :=$(PACKAGE_BASE)/pkg/generated
+CODEGEN_API_PACKAGE :=$(PACKAGE_BASE)/pkg/apis
+CODEGEN_GROUPS_VERSION :=operator:v1alpha1
```

Comment:
We no longer need codegen since we should use `clustercsidrivers.operator.openshift.io` now

Reply:
ack